autosklearn/util/backend.py

```diff
@@ -33,20 +35,34 @@ def __init__(self,
                  temporary_directory,
                  output_directory,
                  delete_tmp_folder_after_terminate,
-                 delete_output_folder_after_terminate):
-        self._prepare_directories(temporary_directory, output_directory)
+                 delete_output_folder_after_terminate,
+                 shared_mode=False):
+
+        # Check that the names of tmp_dir and output_dir is not the same.
+        if temporary_directory == output_directory \
+            and temporary_directory is not None:
+            raise ValueError("The names of tmp dir and output dir "
+                             "must be different.")
+
         self.delete_tmp_folder_after_terminate = delete_tmp_folder_after_terminate
         self.delete_output_folder_after_terminate = delete_output_folder_after_terminate
+        self.shared_mode = shared_mode
+        # attributes to check that directories were created by autosklearn.
+        self._tmp_dir_created = False
+        self._output_dir_created = False
+        self._prepare_directories(temporary_directory, output_directory)
```

Comment:
Could you please add a line break between the variables and the function calls to make it visually easier to see the difference?